tests/test_lm/test_integration/test_jsrun/test_3/cmds

```diff
@@ -0,0 +1,3 @@
+# Task needs to be assigned node 1, physical core 1, HW thread 8
```

Comment:
"physical core 1, HW thread 8" -- I don't think this is too useful.
"physical cores 1-8, HW thread 1 of each physical core" maybe? * (marked for reference)

Reply:
It means 1 process 8 threads. Neither of your suggestions are correct. I will change all the comments and push